script.js

```diff
@@ -26,7 +26,7 @@ colorApp.paragraphs = {
     orange: 'orange para'
 }
 
-colorApp.endpoint = 'https://x-colors.herokuapp.com';
+colorApp.endpoint = 'https://www.thecolorapi.com/';
```

Comment:
```suggestion
colorApp.endpoint = 'https://www.thecolorapi.com/scheme?';
```
can make the Params section cleaner if we add <scheme> to the endpoint up here